examples/udf.java/zillabase/migrations/000000__create_functions.sql

```diff
@@ -1,4 +1,4 @@
 -- create_functions
 
 CREATE FUNCTION gcd(int, int) RETURNS int
-AS gcd;
+AS 'gcd';
```

Comment:
Is this actually required or we have a parser issue to fix in zilla?

Reply:
I found out that it is not required by risingwave but since we are using antlr gramma from postgres it is required. We have applied all these in other examples just missed it in this example. I think since we are sort of following risingwave we need to relax this rule in our gramma. I will file a ticket if you agree.

Reply:
Seems like relaxing that would make our grammar non-standard which is relevant if we support more than Risingwave at the backend, agree?

Reply:
That's good point. agree.